gitman/shell.py

```diff
@@ -13,18 +13,21 @@
 OUT_PREFIX = "> "
 
 
-def call(name, *args, _show=True, _show_stdout=True, _shell=False, _ignore=False):
+def call(name, *args, _show=True, _stream=True, _shell=False, _ignore=False):
     """Call a program with arguments.
 
     :param name: name of program to call
     :param args: list of command-line arguments
-    :param _show: display the call on stdout
-    :param _show_stdout: display stdout of the call on stdout
+    :param _show: display the call arguments
+    :param _stream: stream realtime output of the call
     :param _shell: force executing the program into a real shell
                    a Windows shell command (i.e: dir, echo) needs a real shell
                    but not a regular program (i.e: calc, git)
     :param _ignore: ignore non-zero return codes
     """
+    if not _show:
+        _stream = False
```

Comment:
The goal here is to avoid passing an extra argument in places where its value can be inferred.